test/commands/sleep_test.exs

```diff
@@ -1,17 +1,56 @@
 defmodule Cog.Test.Commands.SleepTest do
-  use Cog.AdapterCase, adapter: "test"
+  use Cog.CommandCase, command_module: Cog.Commands.Sleep
 
-  @moduletag :skip
+  test "basic sleeping" do
+    request = fn ->
+      new_req(args: [5])
+      |> send_req()
+    end
 
-  setup do
-    user = user("jfrost", first_name: "Jim", last_name: "Frost")
-    |> with_chat_handle_for("test")
+    # We're running the command as a task so it doesn't block the test process
+    Task.async(request)
+    # Then we can yield with a short timeout
+    |> Task.yield(1000)
+    # The timeout should expire before the command returns, resulting in nil
+    |> is_nil()
+    |> assert()
```

Comment:
It might be useful to send a request with a `cog_env` set, spin off a `Task`, assert that yielding to the task in a timeframe shorter than the sleep time results in `nil`, but then yield or await the task and verify that it returns whatever it was supposed to in `cog_env`. That way we can be sure that it was sleeping instead of otherwise failing silently.

Incidentally, that would also cover the "returns what it's passed" test down below.